src/test/java/com/wikia/webdriver/common/core/helpers/ContentLoader.java

```diff
@@ -17,32 +15,28 @@
 
   private ContentLoader() {}
 
+  public static class ContentLoaderException extends RuntimeException {
+    public ContentLoaderException(String reason, Exception ex) {
+      super(reason, ex);
+    }
+  }
+
   /**
    * @param filename - name of file which content should be loaded
    * @return content of desired file
    */
-  public static String loadWikiTextContent(String filename) {
+  public static String loadWikiTextContent(String filename) throws ContentLoaderException {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the declaration of thrown exception 'com.wikia.webdriver.common.core.helpers.ContentLoader$ContentLoaderException' which is a runtime exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)